docs/src/examples/components/List/Variations/ListExampleHorizontal.shorthand.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react'
+import { List, Image } from '@stardust-ui/react'
+
+const items = [
+  {
+    key: 'irving',
+    media: <Image src="public/images/avatar/small/matt.jpg" avatar />,
+    header: 'Irving Kuhic',
+    headerMedia: '7:26:56 AM',
+    content: 'Program the sensor to the SAS alarm through the haptic SQL card!',
+  },
+  {
+    key: 'skyler',
+    media: <Image src="public/images/avatar/small/steve.jpg" avatar />,
+    header: 'Skyler Parks',
+    headerMedia: '11:30:17 PM',
+    content: 'Use the online FTP application to input the multi-byte application!',
+  },
+  {
+    key: 'dante',
+    media: <Image src="public/images/avatar/small/nom.jpg" avatar />,
+    header: 'Dante Schneider',
+    headerMedia: '5:22:40 PM',
+    content: 'The GB pixel is down, navigate the virtual interface!',
+  },
+]
+
+const ListExampleSelectable = () => (
+  <List selectable defaultSelectedIndex={0} items={items} vertical={false} />
```

Comment:
Propose we use `horizontal` instead.  This is because the default list is expected to be vertical and the descriptive alternative is one that is horizontal.

This is similar to why there is an HTML attribute for `disabled` and not `enabled="false"`.

Reply:
Okay, my reasoning was that we already have the `vertical` prop, so users will have to think about before applying `vertical`/`horizontal` on different components, based on how they appear by default...